scripts/run-pr.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
```

Comment:
I left it out on purpose, so that `npm test` can fail and `npm start` still runs, without having something too complicated that plays on exit codes, outputs, ...